pandas/_typing.py

```diff
@@ -32,10 +33,12 @@
 FilePathOrBuffer = Union[str, Path, IO[AnyStr]]
 
 FrameOrSeries = TypeVar("FrameOrSeries", bound="NDFrame")
-Scalar = Union[str, int, float, bool]
+PythonScalar = Union[str, int, float, bool]
```

Comment:
would move DatetimeLikeScalar closer to these (also would add some comments about various sections, e.g. these are Scalars )